artifacts/kube-install/apiserver-list.yaml.template

```diff
@@ -102,7 +102,15 @@ items:
       # kind is normally the CamelCased singular type. Your resource manifests use this.
       kind: NamespaceReservation
 
-# register as aggregated apiserver
+# register as aggregated apiserver; this has a number of benefits:
```

Comment:
May be add in the readme?

Reply:
> May be add in the readme?

Good idea.  He updated the commit to do that too now.